yoyodyne/train.py

```diff
@@ -283,6 +285,17 @@ def get_model(
     scheduler_kwargs = schedulers.get_scheduler_kwargs_from_argparse_args(
         **kwargs
     )
+    separate_features = train_set.config.has_features and arch in [
+        "pointer_generator_lstm",
+        "transducer",
+    ]
+    features_vocab_size = getattr(train_set.index, "features_vocab_size", 0)
+    vocab_size = (
```

Comment:
Should this perhaps be in the `Index`? Or does it need to be here since it depends on the model config (do we want to separate features or not)?